src/knex-tables.d.ts

```diff
@@ -91,7 +91,7 @@ declare module "knex/types/tables" {
     // TODO: Find unknown type
     // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
     breaches_last_shown: null | unknown;
-    all_emails_to_primary: boolean;
+    all_emails_to_primary: boolean | null; // added  null in MNTOR-1368
```

Comment:
Depending on how big of a lift the migration would be: maybe we should rename this to `alert_emails_to_primary`? Since other emails (e.g. the monthly activity email) will always be sent to the primary address, regardless of the value.

(And if the migration is too big of a lift, maybe we want to call that out in a JSDoc comment at least?)

Reply:
Renaming a column can be more complicated than it seems.. this thread gives some practical steps to do so in production: 

https://stackoverflow.com/questions/72278957/how-to-practically-rename-tables-and-columns-in-postgresql-on-a-production-syste

I think in this case, since it doesn't seem like the name is too far off, I'd say the risk outweighs the benefits

I'll add a comment